.idea/gradle.xml

```diff
@@ -1,5 +1,6 @@
 <?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Add it to git ignore. Check the stackoverflow link on how to do this.

https://stackoverflow.com/questions/16736856/what-should-be-in-my-gitignore-for-an-android-studio-project